### ISSUE TITLE
HC query: Groups notifications settings

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#289]
@ilmartyrk I noticed there's notifications for groups in the Figma, which I think we don't have currently - only for topics? 

Wondering if this will be included in phase 1 or only in phase 2? 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/fb35dad2-ff60-4600-8d94-e921f6e15dba)

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/40f0ed26-ee23-4de0-aa18-f0226853e587)

@kevincrepin Am also wondering how managing groups notifications interacts with managing topic ones. I see the notification settings tab is very topic focused at the moment - should groups also be in there? And what trumps what if we do have both - if someone has set their group notifications as one thing but then sets a topic within that to something else - would it be the most recently adjusted thing that sticks, presumably? :) 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/dbd65415-90a9-4486-8037-a6a00f1f77c8)

**Comment by kevincrepin** [comment]
Hmmm, not sure where group notifications went (either in my head or irl :D).

Would need to define what activity you could be notified about (new members joining, leaving, etc) and then it would be either another tab or integrated into this one, grouping topics under groups within the list.

**Comment by BeccaMelhuish** [comment]
@ilmartyrk @anettlinno I imagine developing this would be a "Phase 2" thing?

**Comment by anettlinno** [comment]
Yes @BeccaMelhuish and @kevincrepin, definitely Phase II thing. I added Phase II label to the issue :)
